lv_objx/lv_sw.h

```diff
@@ -176,10 +174,12 @@ lv_style_t * lv_sw_get_style(const lv_obj_t *sw, lv_sw_style_t type);
 
 /**
  * Get the animation time of the switch
- * @param sw pointer to a  switch object
- * @return style pointer to a style
+ * @param sw pointer to a switch object
+ * @return animation time in milliseconds
  */
-uint16_t lv_sw_get_anim_time(const lv_obj_t *sw);
+inline uint16_t lv_sw_get_anim_time(const lv_obj_t *sw) {
```

Comment:
This still isn't `static inline`. Sorry to be picky but it can cause problems.

Reply:
wow, I must be blind. Just fixed this, for real.